app/upgrades/v1/bank_send_list.go

```diff
@@ -0,0 +1,33 @@
+package v1
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func BankSendList(ctx sdk.Context) (ResultList, error) {
+	ctx.Logger().Info(fmt.Sprintf("bank send list:%s", UpgradeName))
+
+	// Specify the JSON file path of the bank send list in the "upgradeBankSendListJson" environment variable.
+	jsonBankSendList := os.Getenv("upgradeBankSendListJson")
```

Comment:
do not file read.
Please describe directly in the source.

Makes it very difficult to reach consensus on a program